### PR TITLE
add runtime .25 to latest-updates

### PR DESCRIPTION
**Pull request by beckykd**

### CODE REVIEW

File: docs/guides/latest-updates.mdx [hunk 1]
```diff
@@ -33,6 +33,26 @@ For more details on any of the topics covered in the blog, be sure to check [the
...
+#### New features
+
+- Jobs now have an [`instance`](/api/qiskit-ibm-runtime/qiskit_ibm_runtime.RuntimeJobV2#instance) property that returns the instance that was used for the job.
+- `default_shots` is now a supported option when using EstimatorV2 in [local testing mode](local-testing-mode).
+
+See the full [0.25.0 release notes](/api/qiskit-ibm-runtime/release-notes#0250-2024-07-02).
+{/* use url format /api/qiskit-ibm-runtime/release-notes#0XYZ to link to the specific 0.XY.Z version of the release notes */}
+
+
 ### Qiskit Runtime IBM Client 0.24.0
```
Comment (Eric-Arellano) [inline docs/guides/latest-updates.mdx:56]: Should this be removed?

Comment (beckykd) [reply docs/guides/latest-updates.mdx:56]: We're discussing that with @javabster

Comment (abbycross) [reply docs/guides/latest-updates.mdx:56]: Consensus: we'll keep the latest 3 months, to be parallel with the SDK releases

**Review by abbycross — APPROVED** [review]
👍

File: docs/guides/latest-updates.mdx [hunk 2]
```diff
@@ -33,6 +33,26 @@ For more details on any of the topics covered in the blog, be sure to check [the
 
 {/* use url format /api/qiskit/release-notes/X.X to link to the specific X.X version of the release notes */}
 
+### Qiskit Runtime IBM Client 0.25.0
+
+*To see the release notes for all versions, visit the [Qiskit Runtime IBM Client release notes](/api/qiskit-ibm-runtime/release-notes).*
+
+**Highlights from the 0.25.0 (03 July 2024) version release**
```
Comment (jyu00) [inline docs/guides/latest-updates.mdx:40]: Here the date is `03 July 2024`, but the date for 0.24 was `2024-06-10`

File: docs/guides/latest-updates.mdx [hunk 3]
```diff
@@ -33,6 +33,26 @@ For more details on any of the topics covered in the blog, be sure to check [the
...
+
+#### Deprecation notes
+
+- The `optimization_level` option in EstimatorV2 is deprecated. Instead, you can perform circuit optimization using the [Qiskit transpiler](transpile) or [Qiskit transpiler service](qiskit-transpiler-service).
```
Comment (jyu00) [inline docs/guides/latest-updates.mdx:44]: Alternatively we can say `EstimatorV2` to indicate we're talking about the class. 

```suggestion
- The `optimization_level` option in Estimator V2 is deprecated. Instead, you can perform circuit optimization using the [Qiskit transpiler](transpile) or [Qiskit transpiler service](qiskit-transpiler-service).
```

**Review by jyu00 — APPROVED** [review]

File: docs/guides/latest-updates.mdx (hunk 2 above)
Comment (beckykd) [reply docs/guides/latest-updates.mdx:40]: yeahhh - I put mine in the style-guidelines-approved format but .24 was in the format that the release notes used.  I'll make them consistent.

**Pull request merged**